CHANGELOG.md

```diff
@@ -1 +1,9 @@
 # Changelog
```

Comment:
Is this autogenerated? Should ti be included?

Reply:
I think this is from merging into the release branch`release-v0.2.0`. You should be able to change the base branch back to `master`.